ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__amd64-nightly.yaml

```diff
@@ -3047,6 +3047,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-disasterrecovery-sanity
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy
+- as: vsphere-ipi-proxy-cnu-f28
+  cron: 45 0 27 * *
+  steps:
+    cluster_profile: vsphere-dis-2
+    env:
+      FEATURE_GATES: '["MachineAPIMigration=true"]'
+      FEATURE_SET: CustomNoUpgrade
+    leases:
+    - env: VSPHERE_BASTION_LEASED_RESOURCE
+      resource_type: vsphere-connected-2-quota-slice
+    test:
+    - chain: openshift-e2e-test-qe-automated-release
```

Comment:
May I know why we are using automated-release chain here?

Reply:
Mistakenly , left it , corrected it now , Thanks.